clipboard/android/src/main/java/com/capacitorjs/plugins/clipboard/Clipboard.java

```diff
@@ -0,0 +1,75 @@
+package com.capacitorjs.plugins.clipboard;
+
+import android.content.ClipData;
+import android.content.ClipDescription;
+import android.content.ClipboardManager;
+import android.content.Context;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.Logger;
+
+public class Clipboard {
+
+    public enum ContentType {
+        STRING,
+        IMAGE,
+        URL
+    }
+
+    private Context context;
+    private String logTag = Logger.tags(this.getClass().getSimpleName());
+
+    public Clipboard(Context context) {
+        this.context = context;
+    }
+
+    public void write(String label, ContentType contentType, String content) {
+        ClipboardManager clipboard = (ClipboardManager) context.getSystemService(Context.CLIPBOARD_SERVICE);
+
+        ClipData data = null;
+        switch (contentType) {
+            case STRING:
+            case IMAGE:
+            case URL:
+                data = ClipData.newPlainText(label, content);
+                break;
+        }
+
+        if (data != null && clipboard != null) {
+            clipboard.setPrimaryClip(data);
+        }
+    }
+
+    public JSObject read() {
```

Comment:
If I were to use this class from Java, I would expect a more type-safe container for clipboard data. This may not be a use case for `JSObject`. Maybe a separate class instead?

Reply:
> I know you didn't write the web part, but we probably shouldn't just copy/paste these implementations. They should be evaluated ad hoc and modernized.

Yeah, thanks for your help with this. This isn't my area of expertise